nats.go

```diff
@@ -827,7 +827,20 @@ func (nc *Conn) ConnectedUrl() string {
 	return nc.url.String()
 }
 
-// Report the connected server's Id
+// ConnectedHost reports the connected server's Host
+func (nc *Conn) ConnectedHost() string {
+	if nc == nil {
+		return _EMPTY_
+	}
+	nc.mu.Lock()
+	defer nc.mu.Unlock()
+	if nc.status != CONNECTED {
+		return _EMPTY_
+	}
+	return nc.url.Host
```

Comment:
To be consistent with the other APIs where we return an URL including scheme, you should maybe include it here.